servicemodels/e2sm_ccc/pdubuilder/cc-action-definition_test.go

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2023-present Intel Corporation
+
+package pdubuilder
+
+import (
+	"testing"
+
+	e2smcccv1 "github.com/onosproject/onos-e2-sm/servicemodels/e2sm_ccc/v1/e2sm-ccc-ies"
+	e2smcommoniesv1 "github.com/onosproject/onos-e2-sm/servicemodels/e2sm_ccc/v1/e2sm-common-ies"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCreateE2SmCCcRIcactionDefinition(t *testing.T) {
+	ricStyleType := &e2smcommoniesv1.RicStyleType{}
+	actionDefinitionFormat := &e2smcccv1.ActionDefinitionFormat{}
+	result, err := CreateE2SmCCcRIcactionDefinition(ricStyleType, actionDefinitionFormat)
+	assert.NoError(t, err)
+	assert.NotNil(t, result)
```

Comment:
It would be nice, if the UT would consist of:
- creating a messahe with PDU builder
- validate the message with `.Validate()`;
- encode the message with recently implemented (in this PR) encoder;
- decode the message;
- compare both encoded and decoded messages and make sure they're equal.
This is a common structure of all unit tests in this repository. Please, align with it.

This comment applies elsewhere to all top-level PDUs.